### ISSUE TITLE
SVD addressBlock usage

### ISSUE THREAD

**Issue by burrbull** [esp-rs/esp32#46]
https://github.com/rust-embedded/svd2rust/runs/3799394000?check_suite_focus=true

```
The following predefined values can be used:

    registers
    buffer
    reserved.
```

https://www.keil.com/pack/doc/CMSIS/SVD/html/elem_peripherals.html#elem_addressBlock

**Comment by MabezDev** [comment]
Thanks for the heads up @burrbull. We're currently working on generating the esp32 SVD in a better way, hopefully we can address this soon.

@jessebraham would you mind taking a look at this, making sure our new svd will be compatible?

**Comment by burrbull** [comment]
I would want to see fast fix for file in svd2rust ci.

Saying about generating SVD, [svd-encoder](https://crates.io/crates/svd-encoder) now released.
So if you use Rust for this, you can create `svd_rs::Device` structure and call `encode`.

**Comment by jessebraham** [comment]
Thanks for letting us know about this. For the time being I have opened a PR which should resolve these issues with the current SVD. I did check out your branch of `svd2rust` to ensure it parsed without errors.

As @MabezDev mentioned, there will be a new SVD coming at some point, which has these 'usage' elements set correctly already.

**Comment by MabezDev** [comment]
Closed via #47, CI should now work @burrbull.
